logger/logger.go

```diff
@@ -0,0 +1,81 @@
+package logger
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"github.com/sirupsen/logrus"
```

Comment:
I know that using existing packages can be easy but how hard is to create our logger ?
I'd like to keep the application footprint very low and importing a lot of external dependencies maybe can increment it.

Reply:
Another alternative that I had in mind is to use [log/slog](https://pkg.go.dev/log/slog), which was recently introduced into golang.  In addition to it, I wanted to also use [tint](https://github.com/lmittmann/tint) as slog.Handler that writes tinted logs.

Reply:
Well actually I was thinking to write our log methods using the native log package.

Reply:
That let us do it. However, I would still suggest to do it in different PR/issue